neo/ProtocolSettings.cs

```diff
@@ -17,6 +17,7 @@ public class ProtocolSettings
         public Fixed8 LowPriorityThreshold { get; }
         public uint SecondsPerBlock { get; }
 
+        public uint StateRootEnableIndex { get; }
```

Comment:
Is this the height from which the state root is used? or what exactly?

Reply:
Yes, it is the height from which the roots are used.